README.md

```diff
@@ -99,6 +99,14 @@ rec.stop()
 
 **stop** will cease capturing audio and disable the monitoring and mic input stream. Will request the recorded data and then terminate the worker once the final data has been published. Will call the `onstop` callback when stopped.
 
+---------
+#### Instance Fields
+
+```js
+rec.encodedSamplePosition
```

Comment:
Why name this `encodedSamplePosition` when the field you are reporting is named `granulePosition`?

Reply:
"granule" appears to be ogg-specific terminology. If you enabled streaming with the wavEncoder (you could also include sample rate conversion), then sample position would be more a generic name I think.